### PR TITLE
Add Rosé Pine Windows Terminal application theme

### PR DESCRIPTION
**Pull request by maxstollmayer**
Also applies the soho vibes to the Windows Terminal application window (e.g. tabs).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -13,16 +13,25 @@
 
 ## Usage
 
-1. In settings, paste the contents of one of the theme files (e.g. `rose-pine.json`) in `schemes`
+1. In settings json, paste the contents of one of the scheme files (e.g. `rose-pine.scheme.json`) in `schemes` and the corresponding theme (e.g. `rose-pine.theme.json`) in `themes`
```
Comment (mvllow) [inline readme.md]: Could we replace "In settings json" with the actual filename? E.g. In `settings.json`

**Review by mvllow — CHANGES_REQUESTED** [review]
The updates look lovely, thank you for taking the time to add window theming. Just one request, then should be good to merge :)

File: readme.md (hunk 1 above)
Comment (maxstollmayer) [reply readme.md]: Thanks for reviewing this so quickly. Sure I can change that but what do you think of the following wording?

## Usage

1. Open `settings.json` from Windows Terminal
2. In the `schemes` section of the file, paste the contents of your chosen scheme file (e.g. `rose-pine.scheme.json`)
3. Navigate to the `themes` section and paste the contents of the corresponding theme file (e.g. `rose-pine.theme.json`).
4. Update `colorScheme` within the `profiles` section to include your chosen scheme:

```json
{
    "profiles": {
        "defaults": {
            "colorScheme": "rose-pine"
        }
    }
}
```

5. Update `theme` to use your chosen theme:

```json
{
    "theme": "rose-pine"
}
```

Comment (mvllow) [reply readme.md]: I like that wording, easier to follow. Below is an example of indenting the code blocks to align with the steps but honestly either option is fine. Code blocks under list items usually change the spacing either way so if you prefer how you have it formatted then no change.

```
1. Step 1
2. Step 2

    ```js
    function add() {}
    ```

3. Step 3
```

1. Step 1
2. Step 2

    ```js
    function add() {}
    ```

3. Step 3

Comment (maxstollmayer) [reply readme.md]: Indenting looks nicer. I included that in the following commit.

**Comment by mvllow** [comment]
Thank you 💜

**Pull request merged**